### DISCUSSION TITLE
Correct way to handle path data?

### DISCUSSION THREAD

**Discussion by Anequit** [AvaloniaUI/Avalonia#7722] [Q&A]
Issue #6230 caught my eye and I loved the first approach from @maxkatz6, but I am confused on how I should handle storing the path data. Should I do a resource dictionary, style sheet with a classes, or is there some other way? Ideally I'd like to be able to just define the path data and reference it like @maxkatz did, but I don't see a "data" type that I could use for something like a resource dictionary key.

![image](https://user-images.githubusercontent.com/51097472/155890595-ca9bd6c3-976b-4494-ba3c-ff6a3e6249bd.png)

Any help or insight on this would be great!

**Comment by amwx** [comment]
You can store it using StreamGeometry
```Xaml
<StreamGeometry x:Key="MyPath">M0,0 L 5,5...</StreamGeometry>

<Path Data="{StaticResource MyPath}" />
```

You should also be able to store the data in a string if you want, but the above approach is probably better
```Xaml
<x:String x:Key="MyPath">M0,0 L 5,5...</x:String>

<Path Data="{StaticResource MyPath}" />
```
